### ISSUE TITLE
Add a "You earned this on ... " to badges

### ISSUE THREAD

**Issue by Krynh** [EDSM-NET/FrontEnd#248]
For badges, add the date when earned to the details.

**Comment by b166er** [comment]
and the system-name with a hyperlink to the planet (if the badge was a galactic-record)!

**Comment by AnthorNet** [comment]
Have you tried clicking on the badges?

**Comment by b166er** [comment]
> Have you tried clicking on the badges?

yes, unfortunately, no further information, just this text: Be first to discover an ...
but where i discovered this and when? no info about it


![image](https://user-images.githubusercontent.com/679114/53508760-ad758900-3aba-11e9-82d4-a0d7ea50018b.png)
